packages/react/App.tsx

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import { CdsAlert, CdsAlertActions, CdsAlertGroup } from './src/alert';
+import { CdsButton, CdsButtonType } from './src/button';
+import { CdsBadge } from './src/badge';
+import { CdsIcon } from './src/icon';
+import { CdsModal, CdsModalActions, CdsModalContent, CdsModalHeader } from './src/modal';
+import { CdsTag } from './src/tag';
+import { ClarityIcons, userIcon, timesIcon } from '@clr/core/icon';
+import '@clr/core/icon/register';
```

Comment:
Since the React `CdsIcon` imports the register this import I don't think is needed